stdlib/3/os/__init__.pyi

```diff
@@ -121,6 +121,8 @@ if sys.version_info >= (3, 6):
 _PathType = Union[bytes, Text]
 
 if sys.version_info >= (3, 6):
+    _PathType = Union[bytes, Text, PathLike]
```

Comment:
@matthiaskramm, reusing private aliases between .pyi files is not that uncommon actually.

@cooperlees, you need to change the alias to be created within `if sys.version_info:else:` as Guido is suggesting.